cmd/kube-scheduler/app/server.go

```diff
@@ -111,7 +111,9 @@ for more information about scheduling and the kube-scheduler component.`,
 		fmt.Fprintf(cmd.OutOrStdout(), "%s\n\n"+usageFmt, cmd.Long, cmd.UseLine())
 		cliflag.PrintSections(cmd.OutOrStdout(), namedFlagSets, cols)
 	})
-	cmd.MarkFlagFilename("config", "yaml", "yml", "json")
```

Comment:
this change does not seem related to the rest of the PR

Reply:
+1. 

@m14815 please remove this change from the PR. The commit message will be inaccurate if there are code as well as language changes.